components/goods/GoodForm.tsx

```
@@ -1,65 +1,64 @@
-                    <ActivityIndicator size="small" />
-                    :
-                    <IconSymbol
-                        name={_id ? "pencil" : "plus"}
-                        color="gray"
-                        size={24} />
-                }
-            </Button>
-            <View className="flex-1">
+        <View className="gap-4">
+            <View className="gap-1">
+                <Text className="text-sm font-medium text-gray-700 dark:text-gray-300">Nom de l'article</Text>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)